src/utils/element-class.js

```diff
@@ -5,6 +5,10 @@ export function addClassToElement(className, element) {
   if (element.classList) {
     element.classList.add(className);
   } else {
+    const classes = element.className.split(' ');
+    if (classes.indexOf(className) > -1) {
+      return;
+    }
```

Comment:
weird event behaviour in IE9 was causing some classes to get added twice, so then not enough of them would get rmeoved. this only adds classes if they don't already exist.